src/main/java/Deadline.java

```diff
@@ -0,0 +1,26 @@
+public class Deadline extends Task {
+
```

Comment:
You could consider removing the unnecessary empty line, and please try to be consistent with the use of empty lines throughout your code